.idea/runConfigurations/Run__env_local_buildConfig_bayern_.xml

```diff
@@ -1,7 +1,7 @@
 <component name="ProjectRunConfigurationManager">
   <configuration default="false" name="Run (env:local+buildConfig:bayern)" type="FlutterRunConfigurationType" factoryName="Flutter">
     <option name="additionalArgs" value="--dart-define=environment=local --flavor Bayern" />
-    <option name="attachArgs" value="--dart-define=environment=local --flavor Bayern" />
```

Comment:
Why did you remove the flavors here? The flavor is required for selecting the proper app name for instance.

Reply:
Flutter attach does not work with flavors (ie. the flavor option is not accepted). Luckily, the attach args can be specified separately (I only adjusted the attach args here).

Reply:
Ah its the attach! Sorry!